examples/problems/1100_callback.ipynb

```diff
@@ -0,0 +1,564 @@
+{
```

Comment:
<p>why do we need the term? i.e. lambda term, src, tgt?</p><p> </p><p>And please provide a description about the signature of the callbacks. i.e. input types and output types, in the description above.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/theislab/moscot_notebooks/pull/70/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='30'/>